docs/index.rst

```diff
@@ -0,0 +1,20 @@
+.. aperturedb-python documentation master file, created by
```

Comment:
is this file automatically generated? If yes, we should not commit it.

Reply:
We modified it from the one which was generated as a default.